consul/check.py

```diff
@@ -246,18 +253,28 @@ def check(self, instance):
             # Make service checks from health checks for all services in catalog
             health_state = self.consul_request(instance, '/v1/health/state/any')
 
+            sc = {}
+            # compute the highest status level (OK < WARNING < CRITICAL) a a check among all the nodes is running on.
             for check in health_state:
+                sc_id = '{0}/{1}/{2}'.format(check['CheckID'], check.get('ServiceID', ''), check.get('ServiceName', ''))
                 status = self.STATUS_SC.get(check['Status'])
                 if status is None:
-                    continue
-
-                tags = ["check:{0}".format(check["CheckID"])]
-                if check["ServiceName"]:
-                    tags.append("service:{0}".format(check["ServiceName"]))
-                if check["ServiceID"]:
-                    tags.append("consul_service_id:{0}".format(check["ServiceID"]))
-
-                self.service_check(self.HEALTH_CHECK, status, tags=main_tags+tags)
+                    status = AgentCheck.UNKNOWN
+
+                self.log.debug("got health_state with status '%s' for sc_id '%s'", status, sc_id)
```

Comment:
I added it to ease the debug from support (instead of asking them to curl the health endpoint from Consul). Since it's on a debug level, it should not impact the customers.